python/pyspark/pandas/missing/frame.py

```diff
@@ -30,8 +30,8 @@ def _unsupported_property(property_name, deprecated=False, reason=""):
 
 
 class _MissingPandasLikeDataFrame:
-    # NOTE: Please update the document "Supported pandas APIs" when implementing the new API.
-    # Documentation path: `python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst`.
+    # NOTE: Please update the pandas-on-Spark reference toc when implementing the new API.
```

Comment:
`toc` -> `document`?

Reply:
Or `documentation`, since we're using the term `Documentation` below.

Either way looks fine to me.

Reply:
Thanks, just change to `document`, just because `documentation` makes line exceed more than 100.😂